src/main/java/com/cdancy/bitbucket/rest/domain/pullrequest/Reference.java

```diff
@@ -31,11 +31,14 @@
 
     public abstract MinimalRepository repository();
 
+    @Nullable
+    public abstract String displayId();
+
     Reference() {
     }
 
-    @SerializedNames({ "id", "repository" })
-    public static Reference create(String id, MinimalRepository repository) {
-        return new AutoValue_Reference(id != null ? id : "refs/heads/master", repository);
+    @SerializedNames({"id", "repository", "displayId"})
+    public static Reference create(String id, MinimalRepository repository, String displayId) {
```

Comment:
Id and displayId isn't same value from api. Exemple for master branch : 
id: "refs/heads/master",
displayId : "master"